src/util/webhook-utils.ts

```diff
@@ -71,6 +71,7 @@ export const emitWebhookProcessedMetrics = (
 			{ err },
 			"Failed to send webhook processing time metrics."
 		);
+		return undefined;
```

Comment:
/nit - it implicitly returns undefined anyway so dont need to add it?

Reply:
I am indifferent between return; and return undefined;  I opted for the more explicit....maybe there should be a lint rule for it ;)